packages/nvidia_nat_langchain/src/nat/plugins/langchain/callback_handler.py

```
@@ -52,11 +54,27 @@ def _extract_tools_schema(invocation_params: dict) -> list:
             try:
                 tools_schema.append(ToolSchema(**tool))
             except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)